README.md

```diff
@@ -1,98 +1,206 @@
-# Orbit MVI for Kotlin and Android
+# Orbit 2 for Kotlin and Android
 
 [![CI status](https://github.com/babylonhealth/orbit-mvi/workflows/Android%20CI/badge.svg)](https://github.com/babylonhealth/orbit-mvi/actions)
 [![codecov](https://codecov.io/gh/babylonhealth/orbit-mvi/branch/main/graph/badge.svg)](https://codecov.io/gh/babylonhealth/orbit-mvi)
-[![Download](https://api.bintray.com/packages/babylonpartners/maven/orbit/images/download.svg)](https://bintray.com/babylonpartners/maven/orbit/_latestVersion)
+[![Download](https://api.bintray.com/packages/babylonpartners/maven/orbit-core/images/download.svg)](https://bintray.com/babylonpartners/maven/orbit-core/_latestVersion)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE.md)
 
 ![Logo](images/logo.png)
 
-Orbit is an MVI framework for Kotlin and Android which we use at [Babylon Health](https://www.babylonhealth.com).
-
 ![slack logo](images/slack-logo-icon.png) [Join us at the Kotlinlang slack!](https://kotlinlang.slack.com/messages/CPM6UMD2P)
 
-If you do not yet have an account with the Kotlinlang slack workspace, [sign up here](https://slack.kotlinlang.org).
+If you do not yet have an account with the Kotlinlang slack workspace,
+[sign up here](https://slack.kotlinlang.org).
 
-## Why Orbit
+## Overview
 
-Orbit provides the minimum structure possible around your redux implementation
-to make it easy to use, yet leave you open to use RxJava's power.
+Orbit is a simple scaffolding you can build a Redux/MVI-like
+architecture around. We use it in production at [Babylon Health](https://www.babylonhealth.com).
 
-When we created Orbit, we initially looked at other redux libraries out there
-but felt they didn't meet our needs. Some didn't handle Android lifecycle, and
-others had elaborate structured APIs while some provided custom functionality
-that RxJava gives you out of the box.
+We've recently released a much-improved version - Orbit 2.
+[Read here why.](#a-bit-of-history)
+Be advised that while we have not tested it in production yet. However, we have
```

Comment:
this feels a little contradictory with respect to line 18... "We use it in production"